infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -366,7 +363,7 @@ private void trimPrefix() {
         // (i.e. trim only complete segments)
         long endSegment = dataStore.getStartingAddress() / RECORDS_PER_LOG_FILE - 1;
```

Comment:
Can you drop the -1 and just use less than, instead of less than or equal. I think its more clear that way.

Reply:
I think its better to use getStartingSegment and keep the - 1, also make sure that you don't close the first active segment in `closeSegmentHandlers` cause an unnecessary reload.

Reply:
Done, changed to getStartingSegment